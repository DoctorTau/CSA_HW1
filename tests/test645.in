622
-6108 6921 8131 -2 -8439 1287 7559 -9911 6764 -4493 -5433 8025 7085 -8630 6911 -3194 -409 1955 -7009 7740 -6168 -8201 -2796 -1213 8898 -5700 -3651 -3871 8603 5727 -7628 7490 3111 2643 9258 -7713 -319 -4878 5943 7184 5659 -871 -231 -8250 7713 -4148 -7828 258 4014 1033 8377 -9760 -1791 2300 2972 2400 5922 -8367 -9724 8669 3071 3528 5131 -1086 -1694 9280 -1786 1313 -7375 4838 -1335 4418 -9008 -627 2555 416 5450 -6487 6292 7267 -9910 7561 -1670 1194 871 2741 -8549 5526 284 -2070 1801 2722 6733 -554 -4568 -1174 -5295 4577 -2293 355 8748 -7486 -9735 6465 -7385 -1459 4884 4230 9613 7273 -8024 -3302 -3315 2643 -2353 2835 2041 -2095 -2023 5991 7808 -4661 -8409 -7485 2101 -7811 4110 -988 -8244 9150 4006 5977 7344 -9824 -5646 243 4302 -7068 531 2849 -5326 -9716 -3308 -6478 -5540 6357 314 6998 -5161 9339 9051 -3621 -9841 2046 -6811 -6097 8827 1493 -6064 2044 -2507 2652 -3934 632 5820 -3198 -9216 -2340 -6907 4748 1823 -9747 6519 -8759 3651 -2917 -2520 -391 -6156 -9220 -7396 9359 4671 -3289 -7695 8700 -7691 -2546 3428 -2914 2296 6548 -9414 6148 383 9314 -2791 9801 -69 -3647 -1983 5419 -816 9916 -4772 -2663 2396 5344 -2274 -392 920 6806 2828 5190 -9932 -3395 3028 -9893 -3722 1200 -7877 -9272 -9370 2418 -3931 -5307 -6723 -7871 -9976 -1550 7551 -8984 -2208 -6906 5178 6372 -6681 7604 5905 -4878 -7718 5293 4949 5281 5708 -4606 2852 -3924 -1301 -2096 5367 -5257 9952 5749 6456 -8047 2212 1573 1206 5641 1170 -9975 -6004 -9577 -8640 9941 -7783 1849 8283 -1595 2457 5 -2445 9192 -3307 5180 4631 -4872 6612 9042 6509 9689 4536 9399 -2576 1703 3876 -3288 -785 -8404 -3338 1117 4298 5319 -7628 -1355 1214 -8387 -5198 2446 3608 7120 5046 -3421 -3086 2141 -1014 779 -498 5917 902 -4525 4091 9442 -5761 5861 -7788 9878 5725 -52 -4531 5273 -2818 4561 1427 -6818 -1698 -8472 -6746 1615 8143 9398 -7643 7244 -2266 9252 9393 6677 6263 -3078 4709 -9320 -3965 -1300 385 6822 9566 -2903 -6404 4998 669 -5979 9930 2888 2821 -4414 -4458 4773 1024 -4784 -4689 8708 6505 -9546 -2256 -2405 1624 5241 2914 1775 6929 1813 9142 -4192 7860 -3077 114 -4363 694 2793 -353 -4113 5010 8025 1755 -9673 -5373 8732 2374 -459 2171 9683 -4240 4245 8669 145 1611 5244 3579 9838 -4655 -7430 3953 -9190 6671 1131 -7715 5849 -9213 7160 -9115 6773 -2358 7340 -1263 1706 -7935 -6899 -6277 -3372 8717 -306 5643 -2216 8775 883 1027 -9313 -7944 -3126 -2294 4203 -7141 -3324 -5818 -6074 6253 -8244 -7805 7730 5540 -9789 5101 -7869 -1630 276 2645 4382 -2932 -5450 -7272 -8331 115 6681 3515 9547 527 -1582 3305 3849 7161 -9970 -3204 -6026 6216 5235 -7008 3362 4782 -4539 -9141 -6545 2131 307 -3947 9564 -3383 -4786 167 -688 -9797 -9992 -1993 -4423 8223 -2705 -2754 -990 -3552 477 -6109 4986 -4025 -7307 5416 2452 2356 336 -1473 922 5735 6836 -3123 6415 -2998 9493 -6201 -5714 2697 1437 5486 2153 905 7861 -2326 5953 8099 707 -9124 -8593 7020 -1547 -4218 -6533 -4382 -8128 1500 -1409 -3304 8920 5445 -3996 5944 -6649 9562 8883 1048 -2943 757 -775 3285 -1228 -634 -392 -5331 2023 5801 2138 6111 4895 -4187 -4535 6160 -8660 8440 909 6862 95 2097 2937 1937 -3670 -8014 4637 6259 -9321 -1555 9378 6466 -6409 -4175 -2141 -7244 -5531 -2572 -9110 -542 -6273 -4098 -6917 -144 -9523 9207 5771 8187 5388 936 -9453 -4643 -9237 8968 1278 -791 3673 -5914 -9490 9305 904 -1973 1506 1971 -1399 -1003 -7120 -7531 5441 5532 -4952 -6658 1126 -8198 -3939 5534 -6825 -8962 8941 2981 212 -4767 5955 -8371 -6384 