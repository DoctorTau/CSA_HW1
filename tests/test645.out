-6108 -2 -8630 -3194 -6168 -2796 -5700 -7628 -4878 -8250 -4148 -7828 -9760 -9724 -1086 -1694 -1786 -9008 -9910 -1670 -2070 -554 -4568 -1174 -7486 -8024 -3302 -988 -8244 -9824 -5646 -7068 -5326 -9716 -3308 -6478 -5540 -6064 -3934 -3198 -9216 -2340 -2520 -6156 -9220 -7396 -2546 -2914 -9414 -816 -4772 -2274 -392 -9932 -3722 -9272 -9370 -9976 -1550 -8984 -2208 -6906 -4878 -7718 -4606 -3924 -2096 -6004 -8640 -4872 -2576 -3288 -8404 -3338 -7628 -5198 -3086 -1014 -498 -7788 -52 -2818 -6818 -1698 -8472 -6746 -2266 -3078 -9320 -1300 -6404 -4414 -4458 -4784 -9546 -2256 -4192 -4240 -7430 -9190 -2358 -3372 -306 -2216 -7944 -3126 -2294 -3324 -5818 -6074 -8244 -1630 -2932 -5450 -7272 -1582 -9970 -3204 -6026 -7008 -4786 -688 -9992 -2754 -990 -3552 -2998 -5714 -2326 -9124 -4218 -4382 -8128 -3304 -3996 -1228 -634 -392 -8660 -3670 -8014 -7244 -2572 -9110 -542 -4098 -144 -5914 -9490 -7120 -4952 -6658 -8198 -8962 -6384 